swatch-tally/deploy/clowdapp.yaml

```diff
@@ -699,7 +699,7 @@ objects:
         and billing_account_id = '_ANY'
         and granularity='DAILY'
         and measurement_type!='TOTAL'
-        AND snapshot_date > now() - interval '2 day'; 
+        AND snapshot_date > date_trunc('day', now() - interval '2 day');
```

Comment:
@kahowell the JIRA description mentioned `snapshot_date=date_trunc('day', now() - interval '2 day')` but I think the correct statement is to use greater than `snapshot_date>date_trunc('day', now() - interval '2 day')`

Reply:
No actually, the point is to export a specific single day's worth of data consistently (the export runs daily). Our daily granularity records use a snapshot date which has a snapshot date that is truncated. Try some queries with gabi (use `count(*)` and/or `select date_trunc('day', now() - interval '2 day');` and you'll see get a good idea data would be returned).

Reply:
What I first did when I started working in this ticket was to import all the data from stage to local and use the query you provided, but I got empty results. 
I did check again and I see now that the snapshot date is sometimes truncated and I guess this is the data we want to export here. 
My problem was that this date was in a different time zone that my now() date, that's why I could not find any records. I've confirmed this is not happening when using gabi.
I updated the pull request with taking into account the time zone which works locally and with gabi. Yet this is only a problem when running the query locally, so let me know if you prefer removing the time zone bits.